.github/workflows/localicious.yaml

```diff
@@ -0,0 +1,37 @@
+name: Update translations
+on:
+  push:
+    paths:
+    - 'EduID/localizations.yaml'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+        cache: 'npm'
+    - name: Install localicious/
+      run: |
+        npm install -g @picnicsupermarket/localicious
```

Comment:
This won't work, because localicious needs to be installed via yarn instead of NPM. You will get an error at the generate part if you do it via NPM

Reply:
It does work: See https://github.com/Tiqr/eduid-app-ios/actions/runs/5485162781/jobs/9993628097

Reply:
Interesting, this was always an issue locally for us. Maybe a difference between mac and linux :)
Great that you got it working!